cpp/include/cugraph/detail/graph_functions.cuh

```diff
@@ -158,35 +158,63 @@ partition_information(raft::handle_t const& handle, GraphViewType const& graph_v
  * Collect all the edges that are present in the adjacency lists on the current gpu
  *
  * @tparam GraphViewType Type of the passed non-owning graph object.
- * @tparam EdgeIndexIterator Type of the iterator for edge indices.
  * @tparam GPUIdIterator  Type of the iterator for gpu id identifiers.
- * @param handle RAFT handle object to encapsulate resources (e.g. CUDA stream, communicator, and
- * handles to various CUDA libraries) to run graph algorithms.
- * @param graph_view Non-owning graph object.
- * @param active_majors_in_row Device vector containing all the vertex id that are processed by
+ * @param[in] handle RAFT handle object to encapsulate resources (e.g. CUDA stream, communicator,
+ * and handles to various CUDA libraries) to run graph algorithms.
+ * @param[in] graph_view Non-owning graph object.
+ * @param[in] active_majors_in_row Device vector containing all the vertex id that are processed by
  * gpus in the column communicator
- * @param active_major_gpu_ids Device vector containing the gpu id associated by every vertex
+ * @param[in] active_major_gpu_ids Device vector containing the gpu id associated by every vertex
  * present in active_majors_in_row
- * @param edge_index_first Iterator pointing to the first destination index
- * @param indices_per_source Number of indices supplied for every source in the range
+ * @param[in] minor_map Device vector of destination indices (modifiable in-place) corresponding to
+ * vertex IDs being returned
```

Comment:
`destination` indices => `minor`_indices assuming that this function doesn't need to know whether the minor is source or destination. If the storage is not transposed, major will be source and minor will be destination, if the graph is stored transposed, then the opposite, right?